src/plugins/workspace/server/plugin.ts

```diff
@@ -136,8 +140,16 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
       this.workspaceConflictControl.wrapperFactory
     );
 
+    const workspaceUiSettingsClientWrapper = new WorkspaceUiSettingsClientWrapper();
+    this.workspaceUiSettingsClientWrapper = workspaceUiSettingsClientWrapper;
     core.savedObjects.addClientWrapper(
       -2,
+      WORKSPACE_UI_SETTINGS_CLIENT_WRAPPER_ID,
+      workspaceUiSettingsClientWrapper.wrapperFactory
+    );
+
+    core.savedObjects.addClientWrapper(
+      -3,
```

Comment:
A meaningful variable to the magical priority number would help, I merged another backport PR and maybe you can follow that pattern.

Reply:
Should we declare the order sequence with a comment if there is?

Reply:
@raintygao right, I'll put a comment